include/SDL3/SDL_hints.h

```diff
@@ -3832,37 +3848,47 @@ extern SDL_DECLSPEC const char * SDLCALL SDL_GetHint(const char *name);
 extern SDL_DECLSPEC SDL_bool SDLCALL SDL_GetHintBoolean(const char *name, SDL_bool default_value);
 
 /**
- * Type definition of the hint callback function.
+ * A callback used to send notifications of hint value changes.
+ *
+ * This is called an initial time during SDL_AddHintCallback with the hint's
+ * current value, and then again each time the hint's value changes.
  *
  * \param userdata what was passed as `userdata` to SDL_AddHintCallback()
  * \param name what was passed as `name` to SDL_AddHintCallback()
  * \param oldValue the previous hint value
  * \param newValue the new value hint is to be set to
  *
+ * \threadsafety This callback is fired from whatever thread is setting a
+ *               new hint valueSDL holds a lock on the hint subsystem when calling this callback.
```

Comment:
```suggestion
 *               new hint value. SDL holds a lock on the hint subsystem when calling this callback.
```

Reply:
Fixed in a5e9faf2937d746b6438d1c3849b46d75b66f650.